com.unity.multiplayer.mlapi/Runtime/NetworkVariable/Collections/NetworkSet.cs

```diff
@@ -499,10 +475,21 @@ bool ISet<T>.Add(T item)
             {
                 OnSetChanged(setEvent);
             }
+        }
 
+        /// <inheritdoc />
+        bool ISet<T>.Add(T item)
+        {
+            Add(item);
             return true;
         }
 
+        /// <inheritdoc />
+        void ICollection<T>.Add(T item)
```

Comment:
nit: Maybe use an expression-bodied method to make it SUPER clear this is just a remap... e.g.
```
bool Add(T item)
{
   ..
}

void ICollection<T>.Add(T item) => Add(item);
```
(you also only need the explicit implementation on one of them)

Reply:
Yeah, cool, I can do that on `ICollection`, but I'm stuck on doing this for `ISet` because it has a different return type

Reply:
My code snippet should work. You need to change your implementation to return a bool, and then just ignore it on ICollection, that should meet both ISet and ICollection needs (but is a moot point if you do the suggestion in the other comment thread)